src/frontend/layouts/default.vue

```diff
@@ -1,237 +1,12 @@
 <script setup lang="ts">
-import type { useAuthStore } from '~/stores/auth'
-import { onMounted, onUnmounted, ref, watch } from 'vue'
-import { useRouter } from 'vue-router'
 import Footer from '~/components/Footer.vue'
-
-const router = useRouter()
-const showUserMenu = ref(false)
-const userMenuRef = ref<HTMLElement | null>(null)
-const isStoreReady = ref(false)
-const authStore = ref<ReturnType<typeof useAuthStore> | null>(null)
-const userName = ref('')
-const isAuthenticated = ref(false)
-const isScrolled = ref(false)
-const isMobileMenuOpen = ref(false)
-
-// Watch for auth store changes
-watch([isStoreReady, authStore], ([ready, store]) => {
-  if (ready && store?.user) {
-    isAuthenticated.value = store.isAuthenticated
-    userName.value = store.user.username
-      || (store.user.email ? store.user.email.split('@')[0] : '')
-  }
-  else {
-    isAuthenticated.value = false
-    userName.value = ''
-  }
-}, { immediate: true })
-
-// Handle scroll events
-function updateScroll() {
-  isScrolled.value = window.scrollY > 20
-}
-
-// Safely initialize the auth store
-onMounted(() => {
-  // Add click event listener for click-outside detection
-  document.addEventListener('click', handleClickOutside)
-
-  // Safe initialization of auth store
-  // Using setTimeout to ensure it runs after Pinia and plugins are initialized
-  setTimeout(() => {
-    try {
-      // Try to dynamically import the store
-      import('~/stores/auth').then((module) => {
-        const { useAuthStore } = module
-        authStore.value = useAuthStore()
-        isStoreReady.value = true
-      }).catch((err) => {
-        console.error('Error importing auth store:', err)
-      })
-    }
-    catch (error) {
-      console.error('Could not access auth store in layout:', error)
-    }
-  }, 100)
-
-  // Initialize scroll event listener
-  window.addEventListener('scroll', updateScroll)
-  updateScroll() // Check initial scroll position
-})
-
-onUnmounted(() => {
-  // Remove event listeners
-  document.removeEventListener('click', handleClickOutside)
-  window.removeEventListener('scroll', updateScroll)
-
-  // Clear auth store reference
-  authStore.value = null
-  isStoreReady.value = false
-})
-
-function handleClickOutside(event: MouseEvent) {
-  const target = event.target as HTMLElement
-  if (userMenuRef.value && !userMenuRef.value.contains(target)) {
-    showUserMenu.value = false
-  }
-}
-
-function handleLogout() {
-  if (isStoreReady.value && authStore.value) {
-    authStore.value.logout()
-    isAuthenticated.value = false
-    userName.value = ''
-    router.push('/')
-    showUserMenu.value = false
-  }
-}
-
-const navLinks = [
-  { name: 'Home', href: '/' },
-  { name: 'About', href: '/about' },
-  { name: 'Dashboard', href: '/dashboard' },
-  { name: 'Issue Badges', href: '/issue' },
-  { name: 'Verify', href: '/verify' }
-]
+import Header from '~/components/Header.vue'
```

Comment:
You don't need to import components as they are automatically imported in[Nuxt](https://nuxt.com/docs/4.x/guide/directory-structure/app/components)